rasa_core/policies/embedding_policy.py

```diff
@@ -269,13 +281,22 @@ def _load_visual_params(self, config: Dict[Text, Any]) -> None:
         self.evaluate_every_num_epochs = config['evaluate_every_num_epochs']
         if self.evaluate_every_num_epochs < 1:
             self.evaluate_every_num_epochs = self.epochs
-
         self.evaluate_on_num_examples = config['evaluate_on_num_examples']
 
+    @staticmethod
+    def _load_tf_config(config: Dict[Text, Any]) -> None:
+        """Prepare tf.ConfigProto for training"""
+        return tf.ConfigProto(
+            device_count={'CPU': config['device_count']},
+            inter_op_parallelism_threads=config['inter_op_threads'],
+            intra_op_parallelism_threads=config['intra_op_threads'],
+            gpu_options={'allow_growth': config['allow_growth']}
+        )
+
     def _load_params(self, **kwargs: Dict[Text, Any]) -> None:
         config = copy.deepcopy(self.defaults)
         config.update(kwargs)
-
+        self.config = config
         self._load_nn_architecture_params(config)
```

Comment:
self._load_nn_architecture_params(self.config)

Reply:
and similar for the rest

Reply:
ok, I got it.